alibuild_helpers/build_template.sh

```diff
@@ -92,7 +92,11 @@ if [[ ! "$SOURCE0" == '' && "${SOURCE0:0:1}" != "/" ]]; then
   else
     # Folder is already present, but check that it is the right tag
     cd $SOURCEDIR
-    git checkout "${GIT_TAG}"
+    if ! git checkout "$GIT_TAG"; then
+      # If we can't find the tag, it might be new. Fetch tags and try again.
+      git fetch -f "$SOURCE0" --tags
```

Comment:
Why not fetching just that specific tag?

Reply:
ping

Reply:
We could fetch just that tag, I don't really mind either way.

Reply:
Done.

Reply:
Testing the new command, it seems that it fetches all new tags anyway:
```
$ git fetch -f upstream "refs/tags/v1.7.3:refs/tags/v1.7.3" 
From https://github.com/alisw/alibuild
 * [new tag]         v1.7.4.rc2 -> v1.7.4.rc2
```